4_gap_analysis/src/Con_byState.R

```diff
@@ -1,95 +1,103 @@
-# This file contains a first cut at partitioning whatever data are currently in
-# your 1_wqpdata/out/data folder into these scenarios:
-#   A (ideal) - sites with 12 samples per year, dissolved P, TP, and NO3
-#   B (almost there, need more constituents) - sites with 12 samples per year, 1 or two of dissolved P, TP, NO3
-#   C (almost there, need more frequency) - sites with 6 samples per year, dissolved P, TP, NO3
-# (as you'll see, I've had to back the number of constituents down to 2 for A
-# and C to get any sites for AL)
+# GAP analysis script for the water quality coordination effort
 
 library(tidyverse)
 library(feather) # install with install.packages('feather')
 library(scipiper) # install with devtools::install_github('USGS-R/scipiper')
+library(XLConnect) # read xlsx files
+
+#***********************
+# batch mode or manually
+runMode<-'man'
+if (runMode=='batch'){
+  setwd('D:/abock/NAWQA/WQR/nawqa_wqp') # Set locally for running in batch mode for quick processing of all states
```

Comment:
it'd be great to use relative file paths here if you can, which is convenient if you use an RStudio project. Then anytime you open the project, your working directory will automatically be set to the nawqa_wqp folder